### PR TITLE
response: addition of etag header

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: tests/test_invenio_workflows_serializer_response.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        rec_serializer = workflow_responsify(
+            TestSerializer(), 'application/json')
+        rec = MagicMock(
+            model=MagicMock(modified=datetime.now()), title='Jessica Jones')
+        resp = rec_serializer(rec)
+
+        assert resp.status_code == 200
+        assert resp.content_type == 'application/json'
+        assert resp.get_data(as_text=True) == "Jessica Jones"
+        assert 'Etag' in resp.headers
```
Comment (david-caro) [inline tests/test_invenio_workflows_serializer_response.py]: Maybe add a couple of tests here, using the flask client, creating a workflow, then checking the etag, updating it and making sure it changed.

Comment (david-caro) [reply tests/test_invenio_workflows_serializer_response.py]: You could also then check sending the `If-None-Match` header thingie and make sure you get the 304 back.

Comment (drjova) [reply tests/test_invenio_workflows_serializer_response.py]: I have to change many thing to make it work with db, I don't know if it worths investing time on that

Comment (david-caro) [reply tests/test_invenio_workflows_serializer_response.py]: okok, maybe we can add something like that to the tests on inspire-next instead.

**Comment by drjova** [comment]
@david-caro shall we first merge this one? And restart travis on next?

**Comment by david-caro** [comment]
:+1:

**Comment by david-caro** [comment]
you might want to pin the version of the package on next too, to make sure

**Pull request merged**